spectacles/cli.py

```diff
@@ -529,6 +533,19 @@ def _build_validator_subparser(
         env_var="LOOKER_GIT_BRANCH",
         help="The branch of your project that Spectacles will use to run queries.",
     )
+    base_subparser.add_argument(
+        "--use-personal-branch",
+        action=EnvVarStoreTrueAction,
+        env_var="SPECTACLES_USE_PERSONAL_BRANCH",
```

Comment:
Out of curiosity, why support as an environment variable?

Reply:
I honestly thought we supported all arguments as environment variables. I now realise we don't. Happy to remove it, but seems useful enough in a CI context to be able to set it as an environment variable once rather than on each validator command.

Reply:
Okay, fine by me.